lib/src/asset_loader.dart

```diff
@@ -1 +1,29 @@
-export 'package:easy_localization/src/asset_loader.dart' show AssetLoader;
```

Comment:
error: The argument type 'CsvAssetLoader' can't be assigned to the parameter type 'AssetLoader'.  AssetLoader in easy_localization and AssetLoader in easy_localization_loader diffrent classes, but same name

Reply:
need back or change type to dynamic in easy_localization package

Reply:
sorry, see changes in PR 'Added `startLocale` #145'